apps/console/src/features/applications/components/settings/attribute-management/attribute-settings.tsx

```diff
@@ -798,11 +1015,16 @@ export const AttributeSettings: FunctionComponent<AttributeSelectionPropsInterfa
     };
 
     useEffect(() => {
```

Comment:
Move useEffects above all the functions.

Reply:
resolved with https://github.com/wso2/identity-apps/pull/3566/commits/66c501e3bafdfeba484d242847e688d8d0ee2611